src/field.rs

```diff
@@ -79,7 +85,7 @@ pub trait FieldElement:
     + DivAssign
     + Neg<Output = Self>
     + Display
-    + From<<Self as FieldElement>::Integer>
+    + TryFrom<usize, Error = FieldError>
```

Comment:
@tgeoghegan just wanted to bump this since you're wanting to be consistent about what primitive types we allow conversion to/from. I think a reasonable choice is `u64`.

Reply:
`TryFrom<u64>` would require writing the implementations outside of the declarative macro, since `impl From<$int> for $elem` is a `u64` conversion already in some cases. However, we could use that `From<u64> for Field64` and write `From<u64> for Field255` to get what we need, without monkeying with the `FieldElement` trait any more.

Reply:
Actually, is `TryFrom<usize>` even necessary? Without this bound, `cargo test --all-features` passes

```suggestion
```